docs/user-guide/modding/initial-setup.md

```diff
@@ -0,0 +1,103 @@
+# Initial Setup
+
+## Setting up your development environment
+
+Before you can start modding, you'll need some tools to actually create mods. Luckily, all of these are **available for free**. There exist some paid options we will cover, but those are primarily aimed at existing developers who already have licenses for these tools. This guide is roughly based on some parts of the [BepInEx setup guide](https://docs.bepinex.dev/articles/dev_guide/plugin_tutorial/1_setup.html), and you can also follow that one if it's clearer.
```

Comment:
```suggestion
Before you can start modding, you'll need some tools to actually create mods. Luckily, all of these are **available for free**. 

>! This guide is roughly based on some parts of the [BepInEx setup guide](https://docs.bepinex.dev/articles/dev_guide/plugin_tutorial/1_setup.html), which is another great resource for learning how to mod
```

this forewarning about paid options seems unnecessary to me - I think it's fine to mention rider below 

I would also add a linebreak and format the bepinex setup guide so it pops out a bit more